### PR TITLE
Add the Headers for Origin on Rest API V1

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#75218](http://central.tri.be/issues/75218)

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | You must have a space after a closing parenthesis | [Global_ID.php:49](https://github.com/moderntribe/tribe-common/blob/e26f003ba230e5d8b3f27164aba8b712eec1bca1/src/Tribe/Utils/Global_ID.php#L49)

> via jenkins/codesniffer-phpcs

File: tests/wpunit/Tribe/Utils/Global_ID_Test.php [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+	/**
+	 * @test
+	 *
+	 * @group utils
+	 */
+	public function it_should_be_instantiatable() {
+		$global_id = new Global_ID;
+
+		$this->assertInstanceOf( 'Tribe\Event_Aggregator\Utils\Global_ID', $global_id );
```
Comment (barryhughes) [inline tests/wpunit/Tribe/Utils/Global_ID_Test.php]: If `$global_id` is a `Global_ID` (alias of pseudo-namespaced `Tribe__Utils__Global_ID`, how can its type be the properly namespaced `Tribe\Event_Aggregator\Utils\Global_ID`?

Comment (bordoni) [inline tests/wpunit/Tribe/Utils/Global_ID_Test.php]: Shoot! I forgot to rename this.

Comment (bordoni) [inline tests/wpunit/Tribe/Utils/Global_ID_Test.php]: #fixed

**Comment by barryhughes** [comment]
🍊

**Pull request merged**